packages/apps/clickhouse/charts/cozy-lib

```diff
@@ -0,0 +1 @@
+../../../library/cozy-lib
```

Comment:
This is not going to work with older versions, because it takes version_map and exports exact directory from commit via `git archive` command

https://github.com/cozystack/cozystack/blob/653e2bc7745d1cc2ef9eb64781b2e5712398f640/packages/apps/Makefile#L8

Reply:
Yes, and that is the intended behavior. Older versions do not use library/cozy-lib as a dependency, so everything is consistent.